lib/github-pages/configuration.rb

```diff
@@ -82,6 +83,11 @@ def development?
         Jekyll.env == "development"
       end
 
+      def merged_defaults
```

Comment:
`defaults_for_env`? `env_default_configuration`? It'd be good to know that this method is sensitive to the environment.